slides/index.html

```diff
@@ -29,7 +29,21 @@ <h1>Coding Dojo</h1>
 					<img src="img/kisspng-ninja-computer-programming-learning-study-skills-avatar.png" width="400" />
 				</section>
 				<section>
-					<h2>What is</h2>
+					<h2>Housekeeping</h2>
+						<p>Restrooms</p>
+						<p>Emergency Exits</p>
+						<p>Keep everything clean</p>
+				</section>
+				<section>
+					<h2>Sponsor</h2>
+						<p>Netscout</p>
```

Comment:
can we not name our sponsor here, else we will have to change this for each venue . What would you suggest ?